.github/workflows/ci.yml

```diff
@@ -13,6 +13,16 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DYNEIN_TEST_NO_DOCKER_SETUP: true
+      # define AWS credentials in environment for test
+      AWS_ACCESS_KEY_ID: "test"
+      AWS_SECRET_ACCESS_KEY: "test"
+    services:
```

Comment:
A memorandum for me: CI of dynein runs directly on the runner machine. Thus, we can access `localhost:<port>` or `127.0.0.1:<port>`. This is described in [docs](https://docs.github.com/en/actions/using-containerized-services/about-service-containers#running-jobs-on-the-runner-machine).